tests/Phinx/Db/Adapter/MysqlAdapterTest.php

```diff
@@ -97,7 +103,7 @@ public function testConnectionWithSocketConnection()
         }
 
         $options = ['unix_socket' => getenv('MYSQL_UNIX_SOCKET')] + MYSQL_DB_CONFIG;
-        $adapter = new MysqlAdapter(MYSQL_DB_CONFIG, new ArrayInput([]), new NullOutput());
+        $adapter = new MysqlAdapter($options, new ArrayInput([]), new NullOutput());
```

Comment:
Wasn't passing the $config with socket configured so not sure if anything is running this test.